CHANGELOG.md

```diff
@@ -52,28 +53,10 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nifgen` (NI-FGEN)
     * #### Added
+          *Added nifgen_trigger.py to demonstrate pulling a trigger from another device.
```

Comment:
Include `example` somewhere in there.

Also, this line should be the only change in this file.